test/Microsoft.AspNet.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -71,9 +71,7 @@ public class BinderTypeBasedModelBinderIntegrationTest
             var modelBindingResult = await argumentBinder.BindModelAsync(parameter, modelState, operationContext);
 
             // Assert
-
-            // ModelBindingResult
-            Assert.Null(modelBindingResult);
+            Assert.False(modelBindingResult.IsModelSet);
```

Comment:
Similar point a few more times in this class. Failures to find data or an appropriate binder aren't quite the same as other failures to bind.